modules/understanding-openshift.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-overview.adoc
+
+:_content-type: CONCEPT
+[id="understanding-openshift_{context}"]
+= Understanding {product-title}
+
+{product-title} is a Kubernetes environment for managing the lifecycle of container-based applications and their dependencies on various computing platforms such as bare metal, virtualized, on-premise, and in the cloud. {product-title} deploys, configures and manages containers. {product-title} offers usability, stability, and customization of its components.
+
+{product-title} utilises a number of computing resources, known as nodes. The nodes has a lightweight, secure operating system based on {op-system-base-full}, known as {op-system-first}.
+
+After a node is booted and configured, it obtains a container runtime, such as CRI-O or Docker, for managing and running the images of container workloads scheduled to it. Kubernetes agent, or kubelet, schedules container workloads on the node. Kubelet is responsible for registering the node with the cluster and receiving the details of container workloads.
```

Comment:
Review at the request of sdudhgao@redhat.com

`
[The] Kubernetes agent, or kubelet, schedules container workloads on the node. [The] kubelet is responsible for registering the node with the cluster and receiving the details of container workloads.
`

Reply:
Addressed